package sec1;

public class ExceptionExam3 {
	public static void main(String[] args) {
		int[] nArr = {90, 85, 100};
		int i = 0;
		while(i<=3) {
			try {
			System.out.println(nArr[i]);
			i++;
			} catch(ArrayIndexOutOfBoundsException e) {
				e.getMessage();
				break;
			}
		}
	}
}
